sonar-core/src/main/resources/org/sonar/core/qualityprofile/db/QualityProfileMapper.xml

```diff
@@ -151,6 +151,14 @@
     </where>
   </select>
 
+  <select id="countProjectsByProfile" resultType="org.sonar.core.qualityprofile.db.QualityProfileProjectCount">
```

Comment:
Yes ! And maybe it should also use INNER JOIN instead of only JOIN (I do not know what is the default behaviour of JOIN)